generation/controlnet/2d_controlnet.ipynb

```diff
@@ -0,0 +1,1190 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    train_loader = DataLoader(train_ds, batch_size=64, shuffle=True, num_workers=4, drop_last=True, persistent_workers=True)
</pre><p>For the CICD issues we can use a really small batch size like 16.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1815/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>